### PR TITLE
Info window implementation!!!

### PR DESCRIPTION
**Pull request by hailin-zhang**
excite

### CODE REVIEW

**Comment by pwesferguson** [comment]
much excite!!! 
you can remove displayPlaces (the console logs) in resultsButtons in this PR if you want

**Comment by hailin-zhang** [comment]
Will update some logic for choosing and then commit

File: imports/ui/pages/results/components/resultsMapContainer.jsx [hunk 1]
```diff
@@ -21,11 +24,11 @@ function decideRandomCount(radius) {
 }
 
 function randomizePlaces(placesArray, count) {
-  let priority = 1;
+  let priority = 0.5;
```
Comment (js1998) [inline imports/ui/pages/results/components/resultsMapContainer.jsx]: Did this fix the problem with some places not showing in the map result?

Comment (hailin-zhang) [reply imports/ui/pages/results/components/resultsMapContainer.jsx]: yep

**Pull request merged**